app/less/index.import.less

```diff
@@ -0,0 +1,44 @@
+.index__wrapp {
```

Comment:
This should be written as nested styles, instead of prefixing each one. This is what will ultimately be output by the compiler, so it is valid code, but nesting the styles will make this file more legible.